docs/RELEASE-NOTES.rst

```diff
@@ -6,15 +6,17 @@ next release
 Added Functionality
 `````````````````````
 * Changed container base image from debian-stretch to debian-buster
-* Support for using AS3 as an orchestration backend for Openshift Routes (Experimental)
+* Support for using AS3 as an orchestration backend for Openshift Routes
```

Comment:
Can we have a very brief list of OpenShift routes support for good readability. 
Edge, Passthrough, re-encrypt and annotations that are supported in this release. More info can go to documentation.

Reply:
That comes user documentation where we specify each feature added and all known issues.

Reply:
Cool. Please give the document link here once its merged.